react-multiple-components/my-app/src/Banner.tsx

```diff
@@ -0,0 +1,6 @@
+type Props = {
+  items: string[];
```

Comment:
It'll likely be easier if this is just a single item instead of a string. Let the RotatingBanner look up which item to display and pass it.